.floki/Dockerfile

```diff
@@ -6,10 +6,10 @@ RUN apt-get -qq update && \
     apt-get -y -qq install wget multiarch-support build-essential libssl-dev && \
     wget -O rustup.sh https://sh.rustup.rs && \
     sh rustup.sh -y && \
-    wget https://downloads.datastax.com/cpp-driver/ubuntu/18.04/dependencies/libuv/v1.35.0/libuv1_1.35.0-1_amd64.deb \
-         https://downloads.datastax.com/cpp-driver/ubuntu/18.04/dependencies/libuv/v1.35.0/libuv1-dev_1.35.0-1_amd64.deb \
-         https://downloads.datastax.com/cpp-driver/ubuntu/18.04/cassandra/v2.15.3/cassandra-cpp-driver_2.15.3-1_amd64.deb \
-         https://downloads.datastax.com/cpp-driver/ubuntu/18.04/cassandra/v2.15.3/cassandra-cpp-driver-dev_2.15.3-1_amd64.deb && \
+    wget https://downloads.datastax.com/cpp-driver/ubuntu/18.04/dependencies/libuv/v1.23.0/libuv1_1.23.0-1_amd64.deb \
```

Comment:
Thi versions number's gone backwards. Is that correct?

Reply:
Yes, this is a library so we want minimal dependencies. So we test with the oldest supported version of each library. The workflow build.yml has this correct, but for some reason floki.yaml was not. This fixes that.

Cassandra has to be 2.16.0 or greater to support the recent additions for Astra (cloud secure connection).

Reply:
Superb, thanks.